### ISSUE TITLE
Usage guide needs improvement

### ISSUE THREAD

**Issue by ensemblebd** [marcuswestin/store.js#281]
Just saying. Including the legacy script or the everything script, and ```store``` is undefined. 
RequireJS can't find it without explicitly specifying the context, unless you happen to throw the project in the root of your website which would be highly strange. 

I've spent 30 minutes just trying to figure out how to use this damn thing. Incredibly frustrating. 
Wish I could say it was a breeze to utilize this, but it's not. 

Just curious if anyone else has had a positive experience with installation? Or is it just me.

**Comment by pavittarx** [comment]
Indeed, I too have issues while exploring documentation. For example, the guide to use Expire pluging, simply sends you to the tests written for testing plugins.

**Comment by yellow1912** [comment]
@pavittarx For expire, I'm not sure if we have to run removeExpiredKeys or not. Have you found out?
